src/main/java/qi/parser/Parser.java

```diff
@@ -0,0 +1,123 @@
+package qi.parser;
+
+import qi.command.AddCommand;
+import qi.command.Command;
+import qi.command.DeleteCommand;
+import qi.command.ExitCommand;
+import qi.command.FindCommand;
+import qi.command.InvalidCommand;
+import qi.command.ListCommand;
+import qi.command.MarkCommand;
+import qi.command.UnmarkCommand;
+import qi.qiexception.QiException;
+
+import java.time.LocalDate;
+
+/**
+ * Processes string input from users
+ */
+public class Parser {
+
+    /**
+     * Returns a suitable command to execute the input of users in
+     * the form of string.
+     *
+     * @param command String representation of the user command.
+     * @return Command to be executed
+     * @throws QiException If the user input does not match the required format.
+     */
+    public static Command parse(String command) throws QiException {
+        if (command.equals("bye")) {
+            return new ExitCommand();
+        }
```

Comment:
I think you can also use case switch or a command map, as multiple if else statements may take up a lot of space